crates/data-generator/src/blocks/output.rs

```diff
@@ -0,0 +1,586 @@
+use indexmap::IndexMap;
+use proc_macro2::TokenStream;
+use quote::ToTokens;
+
+use super::model::Block;
+use crate::name::Name;
+
+pub(super) fn gen_blocks_mod(blocks: IndexMap<Name, Block>) -> TokenStream {
```

Comment:
```suggestion
pub fn gen_blocks_mod(blocks: IndexMap<Name, Block>) -> TokenStream {
```
Unnecessary encapsulation